src/script/lua_api/l_mainmenu.cpp

```diff
@@ -967,6 +967,23 @@ int ModApiMainMenu::l_get_active_renderer(lua_State *L)
 	return 1;
 }
 
+/******************************************************************************/
+int ModApiMainMenu::l_get_active_irrlicht_device(lua_State *L)
+{
+	const char *device_name = [] {
+		switch (RenderingEngine::get_raw_device()->getType()) {
+		case EIDT_WIN32: return "WIN32";
+		case EIDT_X11: return "X11";
+		case EIDT_OSX: return "OSX";
+		case EIDT_SDL: return "SDL";
+		case EIDT_ANDROID: return "ANDROID";
+		default: return "Unknown";
```

Comment:
Is it possible to translate `"Unknown"` (either here or in `tab_about.lua`)?

Reply:
I could make it translatable. Should I?
I didn't yet because it's names, apart from the "Unknown", which actually doesn't ever appear in practice.

Reply:
If so, it can be left as it is.